tests/config.c

```diff
@@ -308,13 +332,45 @@ config_parse(master_config *cfg, const uint8 num_config, int argc, char *argv[])
                cfg[cfg_idx].verbose_progress = TRUE;
             }
          }
+         /*
+          * Arguments to run Splinter configured with shared memory.
+          */
+         config_has_option("use-shmem")
+         {
+            for (uint8 cfg_idx = 0; cfg_idx < num_config; cfg_idx++) {
+               cfg[cfg_idx].use_shmem = TRUE;
+            }
+         }
+         config_set_mib("shmem-capacity", cfg, shmem_size) {}
+         config_set_gib("shmem-capacity", cfg, shmem_size) {}
```

Comment:
some functional and unit-tests that use `config_parse()` interface and are enhanced to run with `--use-shmem` can be run with diff configurations of shared memory size.

Going forward, this will be specifically useful for functional `splinter_test` and also for perf tests.